src/Nethermind/Nethermind.Facade/BlockchainBridge.cs

```diff
@@ -169,13 +170,14 @@ public Keccak SendTransaction(Transaction tx, TxHandlingOptions txHandlingOption
 
         public TxReceipt GetReceipt(Keccak txHash)
         {
-            var txReceipt = _receiptStorage.Find(txHash);
-            if (txReceipt != null)
+            var blockHash = _receiptFinder.Find(txHash);
```

Comment:
strange API -> receipt fider Fin(keccak) return blockhash?

Reply:
Yes its a bit abstraction leakage I found myself in, let me rename it